WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewView.swift

```diff
@@ -42,55 +38,44 @@ struct WooPaymentsDepositsCurrencyOverviewView: View {
                 }
 
                 if isExpanded {
+                    Text(viewModel.balanceTypeHint)
+                        .font(.footnote)
+                        .foregroundColor(.secondary)
+                        .padding(.vertical, 8)
                     Divider()
-
-                    HStack(alignment: .top) {
-                        Image(systemName: "calendar")
-                            .accessibilityHidden(true)
-                        Text(viewModel.balanceTypeHint)
-                            .font(.footnote)
-                    }
-                    .foregroundColor(.secondary)
-                    .padding(.vertical, 8)
                 }
             }
 
             if isExpanded {
-                Text(Localization.depositsHeader.localizedUppercase)
+                Text(Localization.lastDepositHeader.localizedUppercase)
                     .font(.subheadline)
                     .foregroundColor(.secondary)
                     .frame(maxWidth: .infinity, alignment: .leading)
                     .accessibilityAddTraits(.isHeader)
-                LazyVGrid(columns: [GridItem(.flexible(maximum: 60), alignment: .leading),
-                                    GridItem(alignment: .leading),
+                LazyVGrid(columns: [GridItem(.flexible(maximum: 44), alignment: .leading),
                                     GridItem(.flexible(minimum: 100), alignment: .leading),
+                                    GridItem(alignment: .leading),
                                     GridItem(alignment: .trailing)],
                           spacing: 16) {
-                    Text(Localization.nextDepositRowTitle)
-                    Text(viewModel.nextDepositDate)
-                    WooPaymentsDepositsBadge(status: viewModel.nextDepositStatus)
-                    Text(viewModel.nextDepositAmount)
-
-                    Text(Localization.lastDepositRowTitle)
-                        .foregroundColor(.secondary)
+                    Image(systemName: "calendar")
+                        .accessibilityHidden(true)
                     Text(viewModel.lastDepositDate)
-                        .foregroundColor(.secondary)
+                        .foregroundColor(.primary)
```

Comment:
We should group these in a column together by wrapping an HStack around them. There's no need to keep the icon in a separate column.

Reply:
Good point, it made sense before when we had more elements but not now. Addressed here https://github.com/woocommerce/woocommerce-ios/pull/11533/commits/c46153f97b6e1ef510e2cb75eb1363fad8b829fc